pkg/syndesis/action/initialize.go

```diff
@@ -34,8 +38,14 @@ func (a *Initialize) Execute(syndesis *v1alpha1.Syndesis) error {
 		target.Status.Reason = v1alpha1.SyndesisStatusReasonDuplicate
 		target.Status.Description = "Cannot install two Syndesis resources in the same namespace"
 		logrus.Error("Cannot initialize Syndesis resource ", syndesis.Name, ": duplicate")
+	} else if syndesisAlreadyInstalled {
+		// One uninitialized Syndesis CR and Syndesis already installed: attaching the installation to the resource
+		target.Status.InstallationStatus = v1alpha1.SyndesisInstallationStatusAttaching
```

Comment:
In what case can this happen ? Having a single fresh Syndesis resource with no state but with an already installed Syndesis ? This could be only the legacy use case, or ? And that is tackled by the legacyController during startup which creates a "Merge" phased resource. Only when in the meantime someone deploys a fresh Syndesis resource, a kind of race could happen.

I don't think we should put too much effort on the legacy case, as this will be a one shot thing. And we should remove that handling for 7.2 imo where we should require, when updating from 7.0 to 7.2 it needs to be done via 7.1

Reply:
Yes, removing it..